src/trace.rs

```diff
@@ -8,8 +8,7 @@ use std::ptr;
 use std::time::Duration;
 
 use super::ffi;
-use crate::error::error_from_sqlite_code;
```

Comment:
This use has been moved into the config_log function since it is now conditionally compiled only when not a loadable extension.